akka-docs/rst/scala/code/docs/actor/FSMDocSpec.scala

```diff
@@ -16,17 +16,17 @@ class FSMDocSpec extends MyFavoriteTestFrameWorkPlusAkkaTestKit {
 
   //#fsm-code-elided
   //#simple-imports
-  import akka.actor.{ Actor, ActorRef, FSM }
+  import akka.actor.{ ActorRef, FSM }
   import scala.concurrent.duration._
   //#simple-imports
   //#simple-events
   // received events
-  final case class SetTarget(ref: ActorRef)
-  final case class Queue(obj: Any)
+  case class SetTarget(ref: ActorRef)
+  case class Queue(obj: Any)
   case object Flush
 
   // sent events
-  final case class Batch(obj: immutable.Seq[Any])
+  case class Batch(obj: Seq[Any])
```

Comment:
`Seq` *is* in fact mutable, see here for an in depth article on it: http://hseeberger.github.io/blog/2013/10/25/attention-seq-is-not-immutable/

This is why in the Akka codebase we type `immutable.Seq` explicitly, because sometimes we really mean the mutable one.